scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -12,156 +12,260 @@ output:
+Interestingly, it looks like samples from the ARMS and ERMS subtypes tend to cluster with samples of the same subtype rather than cluster all together. 
 
-We can also look at the distribution of cell types to see if these subdiagnoses have different or shared cell types.
+In the integration notebook we also looked at the distribution of cell types after integration.
+In that notebook we discussed that cells of the same cell type are expected to integrate with other cells of the same type.
+Is that the case with this dataset?
 
-*Talk about looking across cell types and comparing cell types* 
+A word of caution when evaluating the cell type results for this dataset: The cell types for this dataset were assigned in a two stage process as described in [Patel _et al._ (2022)](https://doi.org/10.1016/j.devcel.2022.04.003).
+The first stage assigned cells as tumor or non-tumor.
+The next stage further classified tumor cells into type of tumor cell (i.e., myoblast, myocyte, or mesoderm).
+Some samples could not be further classified, so all of their tumor cells are denoted `Tumor`.
+The samples which could be further classified have a mix of `Tumor_Mesoderm`, `Tumor_Myoblast`, and `Tumor_Myocyte`.
 
-```{r}
-# facet the UMAP by sample to show the distribution of cell types across each subdiagnosis
+```{r celltype UMAP}
+# # UMAP of all libraries, separating by diagnosis group and labeling cell type
 scater::plotReducedDim(integrated_sce,
                        dimred = "fastmnn_UMAP",
+                       # color each point by cell type
+                       colour_by = "celltype_broad",
+                       point_size= 0.5, 
+                       point_alpha = 0.4)
+```
+
+That doesn't appear to be the case with this dataset. 
+Unlike with the previous datasets we have seen where all cells of the same cell type always grouped together, this dataset shows some slightly different patterns and not all cells of the same cell type cluster together.
+One reason is that tumor data can be heterogeneous and every tumor is unique.
+Depending on the tumor type we may not expect every sample to integrate perfectly.
+In this particular case we are looking at two subtypes of RMS that have been shown to have distinctly different mutation burdens and differentiation states, so it's likely that those differences are contributing to how well they will be integrated.
+
+To confirm our hypothesis that cells are grouping together both by cell type and by RMS subtype we can create a plot that incorporates both pieces of metadata.
+We will take advantage of the `facet_wrap()` function from `ggplot2` to look at two (or more, but we'll start with two) variables in the `colData` at once - the cell type and the subdiagnosis.
+In the below plot we will color our cells by cell type but we will also use `facet_wrap()` to plot the cells from each subdiagnosis in their own separate panel of the plot.
```

Comment:
```suggestion
In the below plot we will color our cells by cell type while also using `facet_wrap()` so that cells from different subdiagnoses will be in their own plot panel.
```